epersonregistrations.md

```diff
@@ -0,0 +1,58 @@
+# EPerson registration Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**GET /api/eperson/epersonregistrations**
```

Comment:
While I don't feel strongly either way, we might consider simplifying this to just `/api/eperson/registrations`

Reply:
I did base the name on the current code, but the term eperson is indeed already present in the category.
Adjusted now